src/content/docs/business-observability/touchpoints.mdx

```diff
@@ -0,0 +1,225 @@
+
+import threshold from 'images/business-observability-screenshot-touchpoint-threshold.png'
+
+import newThreshold from 'images/business-observability-screenshot-tune-update-touchpoint.png'
+
+import turnOffTouchpoint from 'images/business-observability-screenshot-turn-off-touchpoint.png'
+
+import offTouchpoint from 'images/business-observability-screenshot-off-touchpoint.png'
+
+import nrqlQuery from 'images/business-observability-screenshot-touchpoint-nrql-query.png'
+
+Touchpoints are the more granular entities within the PathPoint model and behave similarly to a specific browser application or APM (Application Performance Monitoring). Each step of a Pathpoint is connected to one or more touchpoints, and the status of every stage is determined by the culmination of its respective touchpoints.
+
+Since touchpoint is the pulse of each Pathpoint, it is crucial to test, tune, and work with Touchpoint Queries to obtain the desired data accurately.
+
+On this page, you learn to:
+
+- Test a touchpoint
+- Tune touchpoint thresholds
+- Run touchpoint queries
+- Turn off touchpoint
+
+### Test a touchpoint
+
+Pathpoint offers the functionality to test the validity and health of a specific touchpoint.
+
+To test a touchpoint:
+
+<Steps>
+
+<Step>
+
+Select a touchpoint to test. Right click on it and select **Queries**.
+
+<img
+  title="Example touchpoint query"
+  alt="Example touchpoint query"
+  src={exampleTouchpoint}
+/>
+
+It opens a dialogue box window.
+
+</Step>
+
+<Step>
+
+Here, click **Test**.
+
+<img
+  title="Test touchpoint"
+  alt="Test touchpoint"
+  src={testTouchpoint}
+/>
+
+</Step>
+
+<Step>
+
+Now you see the result of the test.
+
+<img
+  title="Touchpoint test validated"
+  alt="Touchpoint test validated"
+  src={testValidated}
+/>
+
+In this example, it says **Successfully validated**, meaning that your touchpoint is healthy.
+
+</Step>
+
+If the validation fails, find out why, and fix it, then test it again until you get the **Successfully validated** message.
+
+<Callout variant="important">
+
+When testing a touchpoint, if you get an "Incorrect validated" error, it might be something as simple as because you running the Query on the wrong AccountID. Check to verify that you're running the Query on your AccountID.
+
+<img
+  title="Incorrect validated test because of wrong AccountID"
+  alt="Incorrect validated test because of wrong AccountID"
+  src={incorrectValidated}
+/>
+
+</Callout>
+
+</Steps>
+
+### Tune touchpoint thresholds
+
+One way to get better data from your touchpoints is to tweak their Tuning configuration. It is important to note that the initial values you see during the tuning process are defined by the JSON file uploaded to your Pathpoint. However, it is possible to manually adjust these values later as well.
+
+To tune a touchpoint:
+
+<Steps>
+
+<Step>
+
+Select a touchpoint to test. Right click on it and select **Tune**.
+
+<img
```

Comment:
Since we have two screenshots so close together, maybe we could just show the **Tune** dialog box.